src/formatDate.js

```diff
@@ -8,7 +8,44 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const INPUT_SYMBOL = fromFormat.pop();
```

Comment:
Consider using `const` instead of `pop()` to access the last element of `fromFormat` array. Using `pop()` modifies the array, which is not necessary and can lead to unintended side effects if the array is used elsewhere after this function call.